tap_github/client.py

```diff
@@ -137,7 +138,7 @@ class GithubClient:
     def __init__(self, config):
         self.config = config
         self.session = requests.Session()
-        self.base_url = "https://api.github.com"
+        self.base_url = config.get('api_endpoint', DEFAULT_DOMAIN)
```

Comment:
```suggestion
        self.base_url = config.get('base_url', DEFAULT_DOMAIN)
```

Reply:
Also need to update the readMe file as well.

Reply:
Updated readme file and client class attribute `self.base_url`.